chrome/scripts/popup.js

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Usually I'd just put this button in the html file, it's good that I'm seeing a different way of doing what I do. Are there like specific advantages that this approach has?

Reply:
Actually yes. I had done it that way before and it kept giving me errors. Something about restrictions to multiline code to prevent browsers getting attacked by hackers.

Reply:
Oh I see